### DISCUSSION TITLE
Who is using Nitro?

### DISCUSSION THREAD

**Discussion by pi0** [nitrojs/nitro#1015] [General]
So today I was randomly checking [npm depended](https://www.npmjs.com/browse/depended/nitropack) page and discovered some cool ideas being built around Nitro 🚀  

I made this discussion to keep listing projects as being discovered. If you are thinking to make a framework / project based on Nitro, or already have one and want to showcase it, this is the place!

### Nuxt

> Nuxt is an intuitive and extendable way to create type-safe, performant and production-grade full-stack web apps and websites with Vue 3.

- Website: https://nuxt.com
- Repository: https://github.com/nuxt/nuxt
- Made by: Nuxt Team

### Stacks

> Headless full-stack framework for Artisans. Develop modern clouds, apps & framework-agnostic libraries, faster. 

- Website: 
- Repository: https://github.com/stacksjs/stacks
- Made by: @brianzzzasd

### Svarta

> Typescript REST API framework

- Website: https://svartajs.github.io/
- Repository: https://github.com/svartajs/svarta

### Servite

> A vite plugin for React SSR / SSG / CSR / Islands

- Website: https://servite.vercel.app/
- Repository: https://github.com/Codpoe/servite
- Made by: @Codpoe

### Analog

> The full-stack meta-framework for Angular

- Website: https://analogjs.org/
- Repository: https://github.com/analogjs/analog
- Made by: @brandonroberts and [contributors](https://github.com/analogjs/analog/graphs/contributors)

### Fastify-Nitro

> Experimental Nitro (unjs/nitro) preset to export a Nitro server as middleware for fastify

- Website: -
- Repository: https://github.com/auguwu/fastify-nitro
- Made by: @auguwu

### ~~SolidStart~~

> SolidStart, the Solid app framework

- Website: https://start.solidjs.com/
- Repository: https://github.com/solidjs/solid-start
- Made by: @ryansolid and [contributors](https://github.com/solidjs/solid-start/graphs/contributors)

> [!WARNING]  
> Sadly Solid (vinxi) decided to diverge from Nitro and fork it partially. Until further notice, we don't support any of it. Use at your own risk, and use their support channels if faced with any issues. Sorry for this.

**Comment by TheAlexLichter** [comment]
@pi0 feel free to add my examples from [my talk in 2022](https://github.com/manniL/talk-unjs-vuejs-berlin-2022/blob/main/slides.md) too ☺️

**Comment by peterhirn** [comment]
SolidStart recently announced they will be moving to Nitro's adapters in Beta 2, https://github.com/solidjs/solid-start/discussions/1052

### SolidStart
> SolidStart, the Solid app framework
- Website: https://start.solidjs.com
- Repository: https://github.com/solidjs/solid-start
-  Made by: @ryansolid and [contributors](https://github.com/solidjs/solid-start/graphs/contributors)

**Reply by afonsojramos** [reply]
@pi0 I believe SolidStart (and vinxi) is not using the fork anymore, is that correct?

**Comment by ayo-run** [comment]
Hi all, thanks for working on Nitro!

We're building [**McFly**](https://github.com/ayoayco/McFly), a no-framework framework powered by Nitro. Still very early, and no packages published yet, but the goal is to assist developers in leveraging the web platform and get out of the way. :)

Specifically, we want a way to:
1. create web apps with vanilla custom elements
2. write real .HTML files
3. no frameworks or reactivity libraries on the browser
4. server-side pre-rendering
5. control on when and where JS is downloaded for interactive elements

See more on our [repo](https://github.com/ayoayco/McFly#readme) or [website](https://mc-fly.vercel.app/)

**Reply by driftyz700** [reply]
People really like reinventing the wheel when it comes to JS frameworks huh

**Reply by ayo-run** [reply]
Yeah they do!

**Comment by atinux** [comment]
We should add SolidStart to the list: https://github.com/solidjs/solid-start/releases/tag/v0.4.0

As well as https://vinxi.vercel.app/

**Comment by InsaneNaman** [comment]
Tanner (creator of tanstack) is about to release tanstack start (based on vinxi). Now, two mainstream projects (SolidStart and Tanstack Start) will be using Nitro under the hood.

**Comment by Lehoczky** [comment]
I've just seen this in the [angular roadmap](https://angular.dev/roadmap#future-work-explorations-and-prototyping), so hyped 🎉 

![image](https://github.com/unjs/nitro/assets/31937175/aee52685-ec75-4daa-9826-bd9c9cf2675d)
